src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -441,9 +443,17 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     case Event(message: CallStartMessage, data) if !data.isPending(message.callKey) =>
       val updatedData = message.handleStatusPersist(this, data)
       val callKey = message.callKey
-
       fetchLocallyQualifiedInputs(callKey) match {
-        case Success(callInputs) => startActor(callKey, callInputs, message.startMode)
+        case Success(callInputs) =>
+          val updateDbCallInputs = globalDataAccess.updateCallInputs(workflow.id, callKey, callInputs)
+          updateDbCallInputs onSuccess {
+            case i =>
+              logger.debug(s"$i call input expression(s) updated in database.")
+              startActor(callKey, callInputs, message.startMode)
+          }
+          updateDbCallInputs onFailure {
+            case e => throw new SQLException(s"Failed to update symbol inputs for ${callKey.scope.fullyQualifiedName}.${callKey.tag}.${callKey.index}", e)
```

Comment:
This shouldn't throw, `self ! AsyncFailure(sqle)` where `sqle` is the exception you've generated here.